core/exec.c

```diff
@@ -2982,9 +2982,9 @@ static lai_api_error_t lai_exec_parse(int parse_mode, lai_state_t *state) {
                         node->type = LAI_NAMESPACE_INDEXFIELD;
                         node->idxf_index_node = index_node;
                         node->idxf_data_node = data_node;
-                        node->idxf_flags = access_type;
-                        node->idxf_size = skip_bits;
-                        node->idxf_offset = curr_off;
+                        node->fld_flags = access_type;
+                        node->fld_size = skip_bits;
+                        node->fld_offset = curr_off;
```

Comment:
This looks broken, we're using both `idxf_` and `fld_` fields even though they belong to different union members.

Reply:
Ah, the union members are actually unified by this commit. Then I'd suggest to rename `idxf_` to `fld_`.